docs/core/usage.qmd

```diff
@@ -309,3 +309,16 @@ subnetwork_id   | Int32
 priority        | Int32
 flow_rate       | Float64
 collect_demands | Bool
+
+## Subgrid levels - `subgrid_levels.arrow`
+
+This table is made for the [IMod Coupler](https://deltares.github.io/iMOD-Documentation/coupler_ribamod_technical.html#ribasim).
+For the basin that has sloping water level, the water level in the basin can be varying.
+Take the varying water level of a basin as input, the level of the basin subgrind are calculated by a linear projection.
+The output are used in the IMod Coupler.
```

Comment:
I would put a reference here to this section: https://deltares.github.io/Ribasim/core/node/basin.html#subgrid
That explains the subgrid functionality.

Whilst this functionality was made for coupling, what is used by the iMOD Coupler is the in-memory subgrid levels, not the results file.

Reply:
I will put the reference there.

Is it the reason that the subgrid-levels.arraow file is empty (only with index name but table is empty)?